tests/test_tracking.py

```diff
@@ -372,3 +388,136 @@ def test_counter(server, dbm_db, tracker_ctx):
     assert ping["api"] == "ping" and ping["seq"] == '1'
     assert hello["api"] == "hello" and hello["seq"] == '2'
     assert sleep["api"] == "sleep" and sleep["seq"] == '2'
+
+
+'''
+The following 7 test cases tests the backward compatibility,
+there are three kinds of client/server : not_tracked
+old_tracked(only support request header not support response header),
+new_tracked(support request and response header)
+'''
+
+
+def test_not_tracked_client_new_tracked_server(server):
```

Comment:
naming could be better:

`test_not_tracked_client_new_tracked_server` => `test_native_client_call_v2_tracking_server`